pandas/io/json/_json.py

```diff
@@ -95,9 +101,19 @@ def to_json(
 
     if lines:
         s = convert_to_line_delimits(s)
+        try:
+            add_new_line = (
```

Comment:
umm, what is all this?

Reply:
Ensure that the file exists and is not empty before adding a new line at the end, so if the file is empty the code skips It. The try part is because errors can raise during the comparison (I copied It from pandas repo Itself, later I try to find the line)